[
 {erldocker, [	
		%% {unixbridge_port, 5000},
		{docker_http, <<"http://127.0.0.1:2375">>}
	     ]},
 {erldns,[
	  {storage,
	   [ {type, erldns_storage_mnesia},
	     {dbname, undefined},
	     {dir, "/opt/veo/dns_data"},
	     {user, undefined},
	     {pass, undefined},
	     {host, undefined},
	     {port, undefined}
	   ]},
	  {servers, [
		     [
		      {name, inet_localhost_1},
		      {address, "0.0.0.0"},
		      {port, 53}, 
		      {family, inet},
		      {processes, 2}]
		    ]},
	  
	  {dnssec, [
		    {enabled, true}
		   ]},
	  
	  {use_root_hints, false},
	  {catch_exceptions, false},
	  {zones, "/etc/veo/zones.json"},
	  {metrics, [{port, 8082}]},
	  {admin,
	   [ {port, 8083},
	     {credentials, {"username", "password"}}
	   ]},
	  {pools, [
		   {tcp_worker_pool, erldns_worker, 
		    [
		     {size, 10},
		     {max_overflow, 20}
		    ]}
		  ]}
	 ]},
 {mnesia,
  [{dir, "/opt/veo/dns_data"}]},

 {sasl,
  [ {sasl_error_logger, false},
    {utc_log, true},
    {error_logger_mf_dir, "/var/log/veo"},
    {error_logger_mf_maxbytes, 10485760},
    {error_logger_mf_maxfiles, 5}
  ]},
 {lager, [
	  {error_logger_hwm, 300},
	  {colored, true},
	  {log_root, "/var/log/veo"},
	  {handlers, [
		      {lager_console_backend, [{level, debug}]},
		      {lager_file_backend, [{file, "error.log"}, {level, error}]},
		      {lager_file_backend, [{file, "console.log"}, {level, info}]},
		      {lager_file_backend, [{file, "debug.log"}, {level, debug}]}
		     ]}
	 ]
 },
 
 {veo, []}
].
